src/sentry/static/sentry/app/views/settings/projectAlerts/projectAlertRulesNew.tsx

```diff
@@ -1,45 +1,46 @@
-import {Params} from 'react-router/lib/Router';
-import {PlainRoute} from 'react-router/lib/Route';
-import PropTypes from 'prop-types';
+import {RouteComponentProps} from 'react-router/lib/Router';
 import React from 'react';
+import styled from 'react-emotion';
 
 import {IssueAlertRule} from 'app/types/alerts';
-import {Location} from 'history';
-import {Panel} from 'app/components/panels';
+import {Organization} from 'app/types';
+import {Panel, PanelBody, PanelHeader} from 'app/components/panels';
+import {SavedIncidentRule} from 'app/views/settings/incidentRules/types';
 import {t} from 'app/locale';
 import AsyncView from 'app/views/asyncView';
 import EmptyStateWarning from 'app/components/emptyStateWarning';
 import PermissionAlert from 'app/views/settings/project/permissionAlert';
-import RuleRow from 'app/views/settings/projectAlerts/ruleRow';
-import SentryDocumentTitle from 'app/components/sentryDocumentTitle';
-import SentryTypes from 'app/sentryTypes';
+import RuleRow from 'app/views/settings/projectAlerts/ruleRowNew';
+import routeTitle from 'app/utils/routeTitle';
+import space from 'app/styles/space';
+import withOrganization from 'app/utils/withOrganization';
 
-type Props = {
-  params: Params;
-  location: Location;
-  routes: PlainRoute[];
-};
+type Props = {organization: Organization} & RouteComponentProps<
+  {
+    orgId: string;
+    projectId: string;
+  },
+  {}
+> &
+  AsyncView['props'];
 
 type State = {
-  ruleList: IssueAlertRule[];
-};
-
-class ProjectAlertRules extends AsyncView<
-  Props & AsyncView['props'],
-  State & AsyncView['state']
-> {
-  static propTypes = {
-    routes: PropTypes.array.isRequired,
-  };
-
-  static contextTypes = {
-    router: PropTypes.object,
-    organization: SentryTypes.Organization,
-  };
+  issueRules: IssueAlertRule[];
+  metricRules: SavedIncidentRule[];
+} & AsyncView['state'];
 
+class ProjectAlertRules extends AsyncView<Props, State> {
   getEndpoints(): [string, string][] {
     const {orgId, projectId} = this.props.params;
-    return [['ruleList', `/projects/${orgId}/${projectId}/rules/`]];
+    return [
+      ['issueRules', `/projects/${orgId}/${projectId}/rules/`],
+      ['metricRules', `/organizations/${orgId}/alert-rules/`],
```

Comment:
This is temporary as we want to combine these into a single endpoint.

Reply:
When the endpoints are joined will rules be sorted as one result set?

Reply:
Yeah